lib/src/pubspec.dart

```diff
@@ -555,6 +576,43 @@ class Pubspec {
         location: location);
   }
 
+  /// Returns a map containing entries of possible typo key found in the
+  /// pubspec mapped to the suggested valid key it might be.
+  Map<String, String> get typoWarnings {
```

Comment:
Maybe this isn't a good place to put it... maybe we should make a `pubspec_typo_validator.dart` and make it also export this functions so we can call it from other places